fbw-a32nx/src/systems/instruments/src/SD/Pages/Cond/Cond.tsx

```diff
@@ -130,7 +130,7 @@ type CondUnitProps = {
 
 const CondUnit = ({ title, trimAirValve, cabinTemp, trimTemp, x, y, offset, hotAir } : CondUnitProps) => {
     const rotateTemp = offset + (trimAirValve * 86 / 100);
-    const overheat = trimTemp > 80;
+    const overheat = trimTemp > 176; // Overheat indicator at 80C, temperature is given in F
```

Comment:
This doesn't make sense. The temp is only F here if you select alternate units. The problem is really that the unit conversions are done too early. Should be at the point they're used in the UI only.

Reply:
You...are absolutely correct, and I'm not sure how I managed to mix that logic up in my head. Somehow I had it flipped and thought it would convert to F if the units were set to C. I'll work on a more complete change which converts the units later.